modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBSelectParentDisplayContext.java

```diff
@@ -68,6 +69,34 @@ public KBSelectParentDisplayContext(
 		_initParentData();
 	}
 
+	public String getContentSelectionTitle() throws PortalException {
+		long kbFolderClassNameId = PortalUtil.getClassNameId(
+			KBFolderConstants.getClassName());
+		long contentSelectionId = getParentResourcePrimKey();
+
+		String contentSelectionTitle;
```

Comment:
Can you rename `contentSelectionTitle` to simply `title`?

Reply:
Alternative, and if possible, can you simply return the title in each of the conditional branches below?
```
if (getResourcePrimKey() == 0) {
    KBFolder kbFolder = ...;

    return kbFolder.getName();
}
else if (...) {
    // similarly with the other branches.
}
...
```

Reply:
I've edited it. Thank you!